app/services/combined_audio_derivative_creator.rb

```diff
@@ -0,0 +1,170 @@
+require 'tempfile'
+
+
+Response = Struct.new(:webm_file, :mp3_file, :fingerprint, :start_times, keyword_init: true)
+#
+# Generate the combined audio derivatives for a
+# given oral history work.
+#
+# some_work = Work.find_by_friendlier_id(friendlier_id)
+# CombinedAudioDerivativeCreator.new(some_work).generate
+#
+# Sample output:
+# <struct Response
+#  webm_file=<File:/path/to/file.webm>,
+#  mp3_file=<File:/path/to/file.mp3>,
+#  fingerprint="e0248c40015d6dc90b0d02937950b5d7",
+#  start_times=
+#   [["ddbd1a8d-c2eb-47b3-85a4-11b4ffb41719", 0],
+#    ["df773502-56d7-4756-a58c-b1f479910e97", 1.593469]]
+#  >
```

Comment:
Might show up when printing as `<File:`, but please make it clear in docs, somehow, that what is returned is a Tempfile. Also make sure there's a test ensuring this.